pages/howto/easybuild.md

```diff
@@ -0,0 +1,374 @@
+
+In this how-to section, we're going to take you through the process to provide you with the information you'll need to be off and running on your own Easybuild installation!
+
+!!! Danger "STOP!"
+    Before going forward, ensure that you've read the documentation on using the [new modules](../software/modules.md).  Are you SURE this software isn't already installed in the latest software release? Use the `module spider` command to search.  You can also check the CCR [GitHub repo](https://github.com/ubccr/software-layer/issues) for open & closed build requests.  Read the information about [building your own software](../software/building.md) and setting up a build environment.  If you're planning to make this software accessible to your entire research group, ensure you've got [this setup](../software/building.md#building-modules-for-your-group) correctly.  
+
+
+Though you don't need to understand everything about an Easybuild recipe file to install software using Easybuild, we do recommend you [read this](https://docs.easybuild.io/writing-easyconfig-files/) and keep it handy to refer to should questions come up while you're editing these recipe files.  
+
+Easyconfig files are named to include the toolchain and version:  
+GNU compilers:  
+[...]-foss-2021b.eb  
+[...]-GCCcore-11.2.0.eb  
+
+Intel compilers:  
+[...]-intel-2022.00.eb  
+[...]-iimpi-2022.00.eb  
+
+As a reminder, these are the [toolchains](../software/releases.md) CCR supports.  You must use one of these when installing software using Easybuild.  If you try a recipe for an unsupported toolchain, Easybuild will attempt to install that toolchain for you.  You don't want this as it will take up too much disk space, take very long to build, and is highly likely that installation will fail. 
+
+**When using Easybuild, do NOT use the CCR login nodes.**  Always use a [compile node](../software/building.md) or do this from a compute node in an [OnDemand desktop](../portals/ood.md#desktop-interactive-apps) session or [interactive job](../hpc/jobs.md#interactive-job-submission).  These installations can use a decent amount of disk space so we recommend you use your project directory for all software installations.  We do not recommend using your home directory as these have small quotas.  If you haven't already done so, create an `easybuild` directory in your group's project directory and set the [Easybuild build prefix](../software/building.md#building-modules-for-your-group) to that directory.  Reminder: this is just an example; you'll need to set the path name to your group's shared project directory, which may not be in `/projects/academic`      
+
+```
+ccruser@vortex ~ $ ssh compile  
```

Comment:
In the examples, remove the user@host prompt. This way when we change the login node hostnames we don't have to update the docs.

Reply:
Removed this throughout doc